Setup page for the conference picture link. (#899)
EuroPython/website: src/content/pages/photos.mdx

---
title: Photos
subtitle: EuroPython 2024 in Pictures
---

# Official Photos

> Check out the official photos from EuroPython 2024! - Enjoy and relive the memories and highlights of the week of Python in Prague & Remote - and share them with your friends and colleagues.

Reviewer: ```suggestion
Check out the official photos from EuroPython 2024! Relive the highlights of the week of Python in Prague & Remote. Don’t forget to share these memories with your friends and colleagues!
```

1. If you want to use -, you should drop the exclamation mark. 

2. There are a lot of "and"s here.  Edited for readability.

<div style={{textAlign: "center"}}>
    <ButtonLink href="https://www.flickr.com/photos/europython/collections/72157723111210737/">Collection of the EuroPython2024 Official Conference Photos</ButtonLink>
</div>

The official conference Photos where taken by:

> - [Jan Kaderka](www.jankaderka.cz)

Reviewer 2: Can we add something like:

```suggestion
The Conference and Social Event photos where taken by:
- [Jan Kaderka](www.jankaderka.cz)

The photos from Sprint days were taken by our community photographers:
- George Zisopoulos
- Moisés Guimaraẽs
```

Reviewer: The URL needs to be https://www.jankaderka.cz/. Otherwise, it doesn't resolve. You can probably edit Laís' comment to do the auto-commit thing.

# Community Photos

> We are collecting community photos from EuroPython 2024. If you have any of YOUR photos you would like to share with the community the process is simple:

Reviewer: ```suggestion
We are collecting community photos from EuroPython 2024. If you have any of YOUR photos you would like to share with the community, the process is simple:
```

---
title: Photos
subtitle: EuroPython 2024 in Pictures
---

# Official Photos
Check out the official photos from EuroPython 2024! Relive the highlights of the week of Python in Prague & Remote. Don’t forget to share these memories with your friends and colleagues!
<div style={{textAlign: "center"}}>
    <ButtonLink href="https://www.flickr.com/photos/europython/collections/72157723111210737/">Collection of the EuroPython2024 Official Conference Photos</ButtonLink>
</div>

> The official conference Photos where taken by:

Reviewer: ```suggestion
```
